lib/glimesh_web/live/chat_live/index.ex

```diff
@@ -59,6 +59,7 @@ defmodule GlimeshWeb.ChatLive.Index do
         |> assign(:show_timestamps, user_preferences.show_timestamps)
         |> assign(:show_mod_icons, user_preferences.show_mod_icons)
         |> assign(:user_preferences, user_preferences)
+        |> assign(:popped_out, session["popped_out"])
```

Comment:
If you give this a default, we can remove a lot of the code changed in the test file and make this component easier to use:
```suggestion
        |> assign(:popped_out, Map.get(session, "popped_out", false))
```